docs/specs/servers/blockchain/blockchain-server-api.md

```diff
@@ -0,0 +1,136 @@
+# Blockchain API Server
+
+## Blockchain RPC
+
+Blockchain RPC API for interacting with different blockchains can be found in a separate `Blockchain RPC` subsection.
+
+## Hexless account profiles API
+
+Email wallet users can register a profile name e.g. `coolname.connect.id` and use it instead of the `0x` address.
+Server API exposes forward lookup for the `name.connect.id` -> `0x` address with chain IDs and reverse lookup for the `0x` -> `name.connect.id`.
+Users can also update the name data including avatar, description text and bind addresses.
+
+### Name lookup
+
+Used to lookup address for the name.
+
+`GET /v1/profile/account/{name}`
+
+* `name` - is the registered name. eg. `coolname.connect.id`
+
+#### Success response body:
+
+```jsonc
+{
+    "name": string,
+    "address": string,
+    "text": (optional)string,
+    "avatar": (optional)string,
+    "registered": datetime,
+    "updated": datetime,
+    "chainIDs" integer[]
```

Comment:
I've changed the response format: 
The address is the same for the Ethereum chainIDs, we just need to register the list that the user allows to work with, so we don't need to duplicate it in the format like:
```
"60": "0x71C7656EC7ab88b098defB751B7401B5f6d8976F",
"1":"0x71C7656EC7ab88b098defB751B7401B5f6d8976F",
```
It's better to expose the list of allowed chainIDs.

Reply:
I think we should postpone other blockchains for the `v2` and stick to the Eth for the MVP and `v1`, because the scope for that is much bigger (different formats, validation, signing, etc.) I'll make the database schema with keeping this in mind so we can add another chains without much effort in it later, but I think we should postpone with this in the server implementation for now.

Reply:
I think it would be easy enough to use CAIP-10 now and should be future-proof, for example the `eip155:1:0xFF` is the account ID. This is how Notify Server does it, and it only supports EVM currently. Assert eip155 prefix to be EVM during signature validation, and then extract the chain ID from that. Can return as an array of CAIP-10 addresses.